ms2/src/org/labkey/ms2/Protein.java

```diff
@@ -51,7 +61,7 @@
 
     private boolean _forCoverageMapExport = false;
 
-    private static final String PEPTIDE_START_TD="<td class=\"%s\" colspan=%d > %s </td>";
+    private static final String PEPTIDE_START_TD="<td class=\"%s\" %s=%s colspan=%d > %s </td>";

```

Comment:
Do we always use "style" for the first new substitution here? If so, can we treat it more like the class substitution here? And we should include quotes on the value regardless.

Reply:
updated